### DISCUSSION TITLE
Higlight tabs that has active/focused subwindow

### DISCUSSION THREAD

**Discussion by nagylzs** [i3/i3#6260] [Q&A]
### Welcome

- [X] Yes, I've searched similar issues and discussions on GitHub and didn't find any.

### Impact

- [ ] This feature requires new configuration and/or commands

### Current Behavior

In tabbed mode, if the tab contains a single window, and it has the focus, then the tab is shown with a different color.

Example:

![image](https://github.com/user-attachments/assets/8b6e30ba-0473-4424-93ac-ba1e33e7ed47)

However, if the tab has been subdivided into multiple windows, and one of them has the focus, then the tab is not different from any other tabs. Example:

![image](https://github.com/user-attachments/assets/e454d102-56bb-463f-ab79-d16416e79666)

On this screenshot, there are multiple tabs, and one of them contains a vertical tile that has a single window in it - namely sublime text. The tab shows the name of the focused windows ( V[sublime_text] ) but it does not have a different color.

The problem with this is that sometimes I want to check which is the tab that I'm currently watching on the screen; but in the above example this is almost impossible. I can read through all tab titles to find out, but of course that is very clumsy, and it may not work in all cases (e.g. when the same program has multiple windows on different tabs).

### Desired Behavior

The tab that has the focused window should be displayed with a different color. Even if the focused window is deeper down in the tile tree.

### i3 version

```text
Binary i3 version:  4.23 © 2009 Michael Stapelberg and contributors
Running i3 version: 4.23 (pid 1863)
Loaded i3 config:
  /home/user/.i3/config (main) (last modified: 2024. máj. 10., péntek, 17:53:15 CEST, 12857956 seconds ago)

The i3 binary you just called: /usr/bin/i3
The i3 binary you are running: i3
```

**Comment by okraits** [comment]
You are not using the default colors. Did you check if using the default colors solves your issue?
For me the split tab has a different color, using the default colors:

![image](https://github.com/user-attachments/assets/46f01175-c6db-4ecc-8564-a945a5bd73e0)

**Comment by orestisfl** [comment]
See `client.focused_inactive` and `client.focused_tab_title`: https://i3wm.org/docs/userguide.html#client_colors

**Comment by nagylzs** [comment]
You are right, I'm sorry for creating the issue. I was using manjaro i3 distribution default settings. It seems they have created a theme with the same color for multiple states:

```
# class                   border  backgr. text    indic.   child_border
  client.focused          #556064 #556064 #80FFF9 #FDF6E3
  client.focused_inactive #2F3D44 #2F3D44 #1ABC9C #454948
  client.unfocused        #2F3D44 #2F3D44 #1ABC9C #454948
  client.urgent           #CB4B16 #FDF6E3 #1ABC9C #268BD2
  client.placeholder      #000000 #0c0c0c #ffffff #000000 
```

I'm not sure why they do that, I'll probably post an issue about this on their site.

Thank you!
